src/services/recovery/__tests__/recovery-state.test.ts

```diff
@@ -157,6 +157,34 @@ describe('recovery-state', () => {
 
         expect(_isMaliciousRecovery({ chainId, version, safeAddress, transaction })).toBe(false)
       })
+      ;['1.0.0', '1.1.1'].forEach((version) => {
+        it(`should return false if the transaction is an official MultiSend call on Safe version ${version} (below the initial MultiSend contract version)`, () => {
+          const chainId = '5'
+          const safeAddress = faker.finance.ethereumAddress()
+
+          const safeAbi = getSafeSingletonDeployment({ network: chainId, version })!.abi
+          const safeInterface = new Interface(safeAbi)
+
+          const multiSendDeployment = getMultiSendCallOnlyDeployment({ network: chainId, version: '1.3.0' })!
```

Comment:
I've extended the tests to check multiple versions in 94d085e62411d64489725ac86ad8d832fcde72a2 but we cannot test 1.4.1 as we are using `@safe-global/safe-deployments` 1.25.0 which doesn't mark 1.4.1 as `released`.

We need to update it in order for the test to work so I've commented it out with a TODO for now. Updating the package to >= 1.26.0 [where it is `released`](https://github.com/safe-global/safe-deployments/releases/tag/v1.26.0) will require extensive testing as it will affect the entire app.